.github/workflows/pr-labeler.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v4
+    - uses: actions/labeler@v5
       if: join(github.event.pull_request.labels) == ''
       with:
         dot: true
```

Comment:
```suggestion
```

Reply:
https://github.com/actions/labeler?tab=readme-ov-file#breaking-changes-in-v5

Reply:
Why is unresolved? `By default, dot input is set to true.`

Reply:
Oops, I think the button hadn't updated on my screen. I had hit resolved, but I guess it had been set between